test/development/acceptance/error-recovery.test.ts

```diff
@@ -954,7 +953,6 @@ describe('pages/ error recovery', () => {
         5 |   throw Error('no ' + i)
         6 | }, 1000)
         7 | export default function FunctionNamed() {
-          :                                       ^
```

Comment:
swc now reports an error for a position that is actually after the last character in the source file (i.e. points at the imaginary EOF character). Seems like the Webpack codeframe printer doesn't handle that? @eps1lon